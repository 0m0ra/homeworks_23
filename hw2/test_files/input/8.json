{
    "Christopher Dixon": {
        "region": "Moscow",
        "registered": "2018-10-01",
        "last_login": "2023-04-02",
        "email": "michaelhill@example.com",
        "age": 65
    },
    "Craig Long PhD": {
        "region": "Paris",
        "registered": "2022-04-06",
        "last_login": "2023-05-14",
        "email": "martinmichelle@example.com",
        "age": 19
    },
    "Rebecca Evans": {
        "region": "London",
        "registered": "2022-05-17",
        "last_login": "2023-05-13",
        "email": "andrewsann@example.com",
        "3": 29
    },
    "Thomas Graham": {
        "region": "Paris",
        "registered": "2022-09-21",
        "last_login": "2023-08-13",
        "email": "sarahkim@example.com",
        "age": 24
    },
    "Austin Williamson": {
        "region": "New-York",
        "registered": "2002-04-01",
        "last_login": "2023-06-06",
        "email": "conniebrown@example.net",
        421: 57
    },
    "Eric Logan": {
        "region": "Saint-Petersburg",
        "registered": "2004-06-17",
        "last_login": "2023-04-20",
        "email": "emily85@example.net",
        "age": 23
    }
}