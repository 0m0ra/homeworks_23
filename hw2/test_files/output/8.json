{"status": "Error", "details": {"msg": "Invalid JSON file provided"}}