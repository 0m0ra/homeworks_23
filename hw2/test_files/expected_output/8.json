{"status": "Error", "details": {"msg": "Invalid JSON file provided (hw2/test_files/input/8.json)"}}